### DISCUSSION TITLE
[Request] Add a "Focus window" block to window controls

### DISCUSSION THREAD

**Discussion by snowboyz0825** [TurboWarp/extensions#1152] [Ideas]
The title says it all. If it can't be added directly to the extension, I would greatly appreciate it even if it was just a single block extension.

**Comment by CubesterYT** [comment]
@snowboyz0825, during V.1 development of Window Controls, I tried implementing this, but it seems only possible in Electron, I'm projecting adding this block during V1.2 of Window Controls. So not anytime soon.
